src/main/java/mate/academy/springboot/datajpa/service/ProductServiceImpl.java

```diff
@@ -34,4 +41,16 @@ public void deleteById(Long id) {
     public void update(String title, BigDecimal price, Category category, Long id) {

```

Comment:
I used save() for update purpose, it works well, and there's no need to pass all the parameters. What to choose, it's up to you.

Reply:
Nice one! I implemented it. Thanks, bro 🤝